java/src/test/java/testDataStructures/testStack.java

```diff
@@ -0,0 +1,101 @@
+package testDataStructures;
+
+import dataStructures.Stack;
+import org.junit.Test;
+
+import java.util.Arrays;
+
+import static org.junit.Assert.*;
+
+public class testStack {
+    @Test

```

Comment:
Another good test: push and pop things on the stack and see if it acts appropriately.